yaml/src/main/java/com/fasterxml/jackson/dataformat/yaml/util/StringQuotingChecker.java

```diff
@@ -17,6 +17,7 @@ public abstract class StringQuotingChecker
     implements java.io.Serializable
 {
     private static final long serialVersionUID = 1L;
+    private static final int SPACE_CODE_POINT = 0x0020;
```

Comment:
While use of magic constants is non ideal this is a single case of well-known ascii code.